modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,9 +90,20 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Import SAML Metadata
+
+You can import your identity provider's SAML 2.0 metadata to Anypoint Platform. You can use *Import IdP Metadata* to import an XML file that contains a SAML 2.0 configuration from your identity provider. The XML file's values populate the *Identity Management SAML 2.0* form.
```

Comment:
I think we can simplify to the following: "You can import your identity provider's SAML 2.0 metadata to Anypoint Platform via the *Import IdP Metadata* link.  Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form."

Reply:
```suggestion
You can import your identity provider's SAML 2.0 metadata to Anypoint Platform using the *Import IdP Metadata* link. Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form.
```